_pages/homepage.md

```diff
@@ -53,93 +53,144 @@ flow:
+
+            ## Linux Enablement (2022 onwards)
+
+            Support for a Morello Linux environment is still under development. 
+
+            ### Initial Proof of Concept Linux environment: Expected end of 2021 (on FVP)
+            * Based around CHERI LLVM/Clang toolchain & Morello ACK.  Provides a prototype (shimmed) musl libC port and hello world userspace example integrated into a standard (A64) rootfs. 
+            * This provides a starting point for incremental enhancements targeting both the Morello hardware and FVP platforms.
+
+            ### Examples of planned work packages, 2022 onwards:
+
+            * Initial Linux environment support for hardware platform (and support for native compilation)
+            * Further evolution of musl lib C support & Pure-cap userspace application ports
+            * Staged introduction of Morello Kernel PCuABI features
+            * Introduction of a Morello GCC/GlibC based environment
+            * Proof of concept work to demonstrate a Morello aware Distro framework
+
+            Further information on the timeline for ongoing work in the Morello Kernel and the use of libshim in C libraries is available below.
+
+            ## CheriBSD
+
+            A mature [CheriBSD](https://www.cl.cam.ac.uk/research/security/ctsrd/cheri/cheri-morello-software.html) environment for Morello is implemented and hosted separately by the University of Cambridge Computing Laboratory. This provides a memory-safe kernel and userspace, along with examples ports of application frameworks.
 
-            This includes initial modifications to a standard (arm64-v8a) AOSP to support capabilities.
-            Including a modified Bionic library and some pure-capability application ports.
       - format: text
         style: bg-white p-4 my-3
         text_content:
           text: |-
-            ## CheriBSD
+            ## Component support & tooling
+
+            ### Firmware
+            Platform ports of standard open source firmware components: SCP firmware, Trusted Firmware TF-A, UEFI EDK II are available for the Morello platform. These are aarch64 platform ports, with some additional low level hardware initialization to enable support of Capabilities in higher level software.
+
+            ### Morello Linux kernel (under development)
+
+            Arm’s work on the Linux Kernel is focused on developing a new kernel-user syscall ABI to support Pure-capability userspace software development. The aim is to replace all pointers at the Kernel-user interface with 129-bit Capabilities, instead of 64-bit integers. This work is linked to development of associated C libraries (Bionic, Musl, GlibC) and developed against a PCuABI (Pure Capability kernel-user ABI) specification. Enabling Android and Linux environments capable of supporting a mix of Pure-capability and “COMPAT” standard 64 bit userspace components. (32bit applications will not be supported)
```

Comment:
You have both "Pure-capability" and "Pure Capability"; be consistent about hyphenation, and don't capitalise them unless at the start of a sentence. Similarly "Capabilities" doesn't deserve a capital letter. 64-bit and 32-bit, not 64 bit and 32bit.

Reply:
Also, the kernel-user ABI is about more than just syscalls, it includes things like the initial process state and signal delivery

Reply:
I would also not say 129-bit and just drop that, the size isn't particularly relevant, and whether they're 128 or 129 bits depends on who you ask and context (and similarly 64-bit integers can just be integers, or "integer virtual addresses", if you want to highlight the important part of the difference)

Reply:
Let's align on using "pure capability" as the long hand form of purecap, capitalised only at start of a sentence.

Ack for change to 64-bit & 32-bit

wrt kernel-user ABI. Let's just delete the word "syscall" form that sentence.

wrt 129-bit: Again Let's just delete that word from the sentence.